src/components/Common/Resizer/Resizer.styl

```diff
@@ -5,10 +5,11 @@
     top 0
     left 100%
     height 100%
-    padding 0 6px
+    padding 0 0
```

Comment:
I believe this padding is for "hit box" for convenient resize, so it's better not to reduce it to zero